spec/expression.dd

```diff
@@ -2810,9 +2811,19 @@ $(H4 $(LNAME2 is-identifier, Identifier Forms))
         is declared to be an alias of $(I Type).
         )
 
+$(SPEC_RUNNABLE_EXAMPLE_COMPILE
+---
+struct S
+{
+    int i;
+}
+static assert(is(typeof(S.i) T) && T.stringof == "int");
```

Comment:
```suggestion
static assert(is(typeof(S.i) T) && is(T == int));
```

Don't rely on the output of `.stringof`, it's implementation defined

Reply:
it's a bug in __traits(identifier)  in first place

```
template temp(T)
{
    pragma(msg, __traits(identifier, T));
}
alias Tint = temp!int; 
```

> Error: argument `int` has no identifier

Reply:
What does `__traits(identifier)` have to do with this code example?

Reply:
> What does `__traits(identifier)` have to do with this code example?

> Don't rely on the output of .stringof, it's implementation defined

Reply:
You don't need a string here. My suggestion uses another `is()` expression.

Reply:
that was not your initial argument, you initial argument was 

> Don't rely on the output of .stringof, it's implementation defined

Reply:
Just because there's no utility for getting a string for built-in types doesn't mean `.stringof` isn't implementation defined anymore, and it doesn't validate this usage, especially considering you don't even need a string in the first place.

Reply:
@dkorpel changed to use `T.sizeof`. I wanted to show that T is valid after any `is` expression.